tests/AbstractErrorHandlerTest.php

```diff
@@ -44,6 +45,34 @@ public function testConstructor()
         }
     }
 
+    /**
+     * @dataProvider errorReportingConstantProvider
+     */
+    public function testHandleErrorShouldNotCapture(bool $expectedToCapture, int $captureAt, int $errorReporting)
```

Comment:
Please move this test under the one for the constructor. As a convention I tried to always have the methods of the tests in the same order as the real methods are declared

Reply:
Isn't already like that?

Reply:
No, there are still a few tests about the constructor that are below this one

Reply:
Done.